pom.xml

```diff
@@ -57,7 +57,7 @@
         <findbugs.version>3.0.2</findbugs.version>
 
         <guava.version>30.1.1-jre</guava.version>
-        <groovy.version>2.5.14</groovy.version>
+        <groovy.version>3.0.9</groovy.version>
```

Comment:
Please also update awaitility.
There is a comment further up in the pom.xml: ```TODO: awaitility 4.0.3+ depends on Groovy 3```

Reply:
Done